app/models/payout.rb

```diff
@@ -62,5 +62,7 @@ def nonprofit_must_be_vetted
 		self.errors.add(:nonprofit, "must be vetted") unless self.nonprofit && self.nonprofit.vetted 
 	end
 
+	def self.publish_created
+		ObjectEvent.create( event_type: 'payout.created')
```

Comment:
This should be `object_events.create(event_type: 'payout.created')`. `object_events` should be a field on Payout. Now that I look at it, it isn't. I'll need to correct that.